core/client/ui/remote-stream.js

```diff
@@ -28,7 +14,7 @@ const checkMediaAvailable = (template, type) => {
         return
     }
 
-    const source = type === streamTypes.screen ? remoteUser.screen?.srcObject : remoteUser.main?.srcObject
+    const source = type === streamTypes.screen ? remoteUserIsNear.screen?.srcObject : remoteUserIsNear.main?.srcObject
```

Comment:
why changing this?

Reply:
remoteUser did not have the needed data to stream, since it's on the array, I could get it like that. Maybe I should name it correctly, it's confusing

Reply:
Oh nice, now I have the good data! Since srcObject is a MediaStream and not stringifiable, it was not passed from modal component on this  commit https://github.com/Refty/lemverse/pull/44/commits/91a08946e46c7be16c3984abbbb730a00bcda0e1. But now it's good, look that's working and can get the object to stream ! Nice